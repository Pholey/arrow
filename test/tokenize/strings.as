// String (str)
"Hello, UTF-8"
"welcome"
"Къеблагъ"
"Welkom"
"Mirë_se_vjen"
"Wellkumma"
"Բարի_գալուս"
"Прывiтанне"
"স্বাগতম"
"ကြိုဆိုပါတယ်"
"歡迎光臨"
"ยินดีต้อนรับ"
"ようこそ"

// Escape sequences
"\\"
"\'"
"\""
"\a"
"\b"
"\f"
"\n"
"\r"
"\t"
"\v"
"\x24"
