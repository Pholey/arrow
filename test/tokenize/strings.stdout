test/tokenize/strings.as:2:1-15: string: Hello, UTF-8
test/tokenize/strings.as:3:1-10: string: welcome
test/tokenize/strings.as:4:1-11: string: Къеблагъ
test/tokenize/strings.as:5:1-9: string: Welkom
test/tokenize/strings.as:6:1-15: string: Mirë_se_vjen
test/tokenize/strings.as:7:1-12: string: Wellkumma
test/tokenize/strings.as:8:1-14: string: Բարի_գալուս
test/tokenize/strings.as:9:1-13: string: Прывiтанне
test/tokenize/strings.as:10:1-10: string: স্বাগতম
test/tokenize/strings.as:11:1-15: string: ကြိုဆိုပါတယ်
test/tokenize/strings.as:12:1-7: string: 歡迎光臨
test/tokenize/strings.as:13:1-15: string: ยินดีต้อนรับ
test/tokenize/strings.as:14:1-7: string: ようこそ
test/tokenize/strings.as:17:1-5: string: \
test/tokenize/strings.as:18:1-5: string: '
test/tokenize/strings.as:19:1-5: string: "
test/tokenize/strings.as:20:1-5: string: 
test/tokenize/strings.as:21:1-5: string: 
test/tokenize/strings.as:22:1-5: string: 
test/tokenize/strings.as:23:1-5: string: 

test/tokenize/strings.as:24:1-5: string: 
test/tokenize/strings.as:25:1-5: string: 	
test/tokenize/strings.as:26:1-5: string: 
test/tokenize/strings.as:27:1-7: string: $
test/tokenize/strings.as:28:1: end
